source/includes/_amp.md

```diff
@@ -1,22 +1,21 @@
-#AMP snippet
+#AMP
```

Comment:
Maybe let's call this section: "Support for AMP" to make it more clear what this section is about?